attestation-service/src/token/ear_broker.rs

```diff
@@ -248,7 +243,7 @@ impl AttestationTokenBroker for EarAttestationTokenBroker {
             let policy_id = format!("{}_{}", policy_ids[0], tee_claims.tee_class);
```

Comment:
Probably not related to this PR. I have tried locally with ear policy naming rules, it is somehow troublesome. I suggest to get rid of coupling the tee_class name with policy name, that means

The `policy_id` given to attestation-service's API is actually the `policy_id` that was just registered. The current behavior is that if we want to verify with policy id `some-id`, we need to register a policy with id `some-id_cpu`, and the backend storage will also show `some-id_cpu.rego`.

Reply:
Are you suggesting that we share one policy between all the different classes? This is possible but it would make the policy really big.

Reply:
Yeah I am open to allowing the caller to specify the policy for each class individually. I didn't do this at first because specifying a policy per class will make the API more complex, but I can see it is a little confusing to append things to the policy. If we can come up with a nice way to change the API, I wouldn't be against it.

Reply:
Yes, I have composed some different ways in the issue https://github.com/confidential-containers/trustee/issues/863